modules/cloudevent-pull-trigger/main.tf

```diff
@@ -41,19 +41,51 @@ resource "google_pubsub_topic_iam_binding" "allow-pubsub-to-send-to-dead-letter"
   members = ["serviceAccount:${google_project_service_identity.pubsub.email}"]
 }
 
+resource "google_storage_bucket" "dlq_bucket" {
```

Comment:
Formatting seems a bit off here.

Reply:
The spaces and such, if you run `tf fmt` it doesn't complain about anything? Like L56 below, I'd expect it to complain:
```
bucket =google_storage_bucket.dlq_bucket[count.index].name
```

because there's no space after the `=`, but maybe I'm just wrong.

Reply:
ah ok, i will check, but that is old code, I have a new one in my local, was running some tests to validate things before pushing it